docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -310,6 +310,19 @@ otherwise the procedure will fail with similar message:
 ``Retention specified (1.00d) is shorter than the minimum retention configured in the system (7.00d)``.
 The default value for this property is ``7d``.
 
+.. _drop-extended-stats:
+
+drop_extended_stats
+~~~~~~~~~~~~~~~~~~~
+
+The ``drop_extended_stats`` command removes extended statistics from metadata files.
+
+``drop_extended_stats`` can be run as follows:
+
+.. code-block:: sql
+
+  ALTER TABLE test_table EXECUTE drop_extended_stats
```

Comment:
```suggestion
  SET SESSION my_catalog.experimental_extended_statistics_enabled = true;
  ALTER TABLE test_table EXECUTE drop_extended_stats
```

Reply:
I believe `drop_extended_stats` isn't protected by the session property. Is it a bug? 
```sql
trino> SHOW SESSION LIKE 'iceberg.experimental_extended_statistics_enabled';
                       Name                       | Value | Default |  Type   |                                                          Description
--------------------------------------------------+-------+---------+---------+--------------------------------------------------------------------------------------------------------------------------------
 iceberg.experimental_extended_statistics_enabled | false | false   | boolean | Allow ANALYZE and use of extended statistics collected by it. Currently, the statistics are collected in Trino-specific format

trino> ALTER TABLE iceberg.tpch.region EXECUTE drop_extended_stats;
ALTER TABLE EXECUTE
 rows
------
(0 rows)
```

Reply:
You're right, it's not.

It doesn't have to be -- it's definitely important to protect the ANALYZE and SELECT code paths, but this one is just helper, but can become protected as well.